themes/red-theme/lms/static/sass/partials/bootstrap/_theme.scss

```diff
@@ -25,3 +25,6 @@ $theme-colors: map-merge((
   light: $gray-100,
   dark: $gray-800
 ), $theme-colors);
+
+// Why is this necessary?
```

Comment:
Good question, why is it?

Reply:
I was unable to find out why the Bootstrap variable didn't pick up the brand color. I think it is a Sass evaluation order issue, but I couldn't figure it out.

Reply:
I think it would be worth leaving a comment here explaining this, so if someone in the future figures out how to resolve the issue they can update the code with more confidence.

Reply:
Done